lib/common/socket.c

```diff
@@ -340,6 +341,11 @@ static void dispose_socket(h2o_socket_t *sock, const char *err)
         sock->_peername = NULL;
     }
 
```

Comment:
```suggestion
```

Nothing more than a nitpick, but it does not make sense to have an empty line here when there's none between `h2o_buffer_dispose(&sock->input);` and `if(sock->_peername != NULL) {`.